lib/web_ui/dev/browser_lock.yaml

```diff
@@ -1,11 +1,21 @@
+## Driver version in use.
+## For an integration test to run, the browser's major version and the driver's
+## major version should be equal. Please make sure the major version of
+## the binary for `chrome` is the same with `required_driver_version`.
+## (Major version meaning: For a browser that has version 13.0.5, the major
+## version is 13.)
+## Please refer to README's `Upgrade Browser Version` section for more details
+## on how to update the version number.
+required_driver_version:
+  ## Make sure the major version of the binary in `browser_lock.yaml` is the

```

Comment:
This no longer needs to refer to `browser_lock.yaml` as everything is in the same file. Instead it should point to the relevant keys in this file that need to be in sync with this value.